File: src/048/p048.py
def solve1() -> int:
    total = 0
    for number in range(1, 1001):
        power = 1
        for _ in range(number):
            power = (power * number) % 10000000000
        total = (total + power) % 10000000000
    return total


def solve() -> int:
    return int(sum(i**i for i in range(1, 1001)) % 10000000000)

File: src/P048/p048.go
package problem048

import "strconv"

func Solve() string {
	total := 0
	for n := 1; n <= 1000; n++ {
		power := 1
		for i := 1; i <= n; i++ {
			power = (power * n) % 10000000000
		}
		total = (total + power) % 10000000000
	}
	return strconv.Itoa(total)
}
